workflow-templates/psalm-matrix.yml

```diff
@@ -44,7 +44,7 @@ jobs:
         run: composer i
 
       - name: Install dependencies
-        run: composer require --dev nextcloud/ocp:${{ matrix.ocp-version }} --ignore-platform-reqs
+        run: composer require --dev nextcloud/ocp:${{ matrix.ocp-version }} --ignore-platform-reqs --with-dependencies
```

Comment:
For me this still fails with:
> Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.

Reply:
https://github.com/nextcloud/announcementcenter/actions/runs/5119157822/jobs/9204060224?pr=665

Reply:
Okay, also doesn't seem to work